pkg/controller.v1/tensorflow/controller.go

```diff
@@ -418,13 +410,8 @@ func (tc *TFController) reconcileTFJobs(tfjob *tfv1.TFJob) error {
 		}
 
 		if tc.Config.EnableGangScheduling {
-			tc.Recorder.Event(tfjob, v1.EventTypeNormal, "JobTerminated", "Job is terminated, deleting PodGroup")
 			if err := tc.DeletePodGroup(tfjob); err != nil {
-				tc.Recorder.Eventf(tfjob, v1.EventTypeWarning, "FailedDeletePodGroup", "Error deleting: %v", err)
 				return err
-			} else {
-				tc.Recorder.Eventf(tfjob, v1.EventTypeNormal, "SuccessfulDeletePodGroup", "Deleted PodGroup: %v", tfjob.Name)
```

Comment:
Why should we remove the event record here? Do you think the events are too much?

Reply:
> Why should we remove the event record here? 

Not remove, just move the code to the `DeletePodGroup` function. 

> Do you think the events are too much?

Yes, for tfjob that is success or failure in every sync, it regenerate the event.

Reply:
Oh got you, LGTM, thanks.